pkg/autoscaler/stats_reporter.go

```diff
@@ -119,6 +123,12 @@ func register() {
 			Aggregation: view.LastValue(),
 			TagKeys:     []tag.Key{namespaceTagKey, serviceTagKey, configTagKey, revisionTagKey},
 		},
+		&view.View{
+			Description: "Current excess burst capacity over average request count in each 60 second stable window",
```

Comment:
How about not mentioning the 60 seconds? If different value is used the description is inaccurate. Same for existing descriptions.

Reply:
works for me. I went for the consistency.
Removed.